housekeeper/add/core.py

```diff
@@ -24,6 +24,7 @@ def add_bundle(self, data: dict) -> models.Bundle:
             bundle_obj = self.new_bundle(name=data['name'], created_at=data['created'])
 
         version_obj = self.new_version(created_at=data['created'], expires_at=data.get('expires'))
+        bundle_obj.versions.append(version_obj)
```

Comment:
... thank goodness no one was using this function :) Otherwise we'd have bundles without versions :(